_authors/chuyuezhang.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Chuyue Zhang
+short_name: Chuyue
+twitter: <twitter handle without the @>
+github: https://github.com/th3ch3wy/
+---
+
+**<Name>** <I like cats>
```

Comment:
Replace `<Name>` with your name and remove the brackets around `<I like cats>` -> `I like cats`